compose/caddy/bin/start-caddy

```diff
@@ -7,6 +7,6 @@ set -eou pipefail
 if [ "$APP_ENV" = "prod" ]; then
     exec /usr/bin/caddy --conf /etc/Caddyfile --log stdout
```

Comment:
if the `-ca` tag is not specified, does it default to using the production letsencrypt server?

Reply:
Yes. I believe so. https://caddyserver.com/docs/automatic-https#testing